### PR TITLE
Updated Earning calculator

### PR DESCRIPTION
**Pull request by ed3ath**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

https://github.com/CryptoBlades/cryptoblades/issues/768

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

(insert description here)

### CODE REVIEW

File: frontend/.env.aw [hunk 1]
```diff
@@ -1,7 +1,8 @@
 VUE_APP_STAKING_ONLY="0"
-VUE_APP_FEATURE_FLAG_RAID="1"
+VUE_APP_FEATURE_FLAG_RAID="0"
 VUE_APP_FEATURE_FLAG_REFORGING="1"
 VUE_APP_FEATURE_FLAG_MARKET="1"
 VUE_APP_FEATURE_FLAG_MARKET_USE_BLOCKCHAIN="1"
 VUE_APP_FEATURE_FLAG_PORTAL="1"
-VUE_APP_API_URL="https://api.cryptoblades.io"
+
+VUE_APP_NETWORK_ID="1337"
```
Comment (kcper) [inline frontend/.env.aw:8]: I don't think we want these changes in main. Correct me if I'm wrong.

File: frontend/src/components/smart/EarningsCalculator.vue [hunk 2]
```diff
@@ -145,7 +145,7 @@ import { getConfigValue } from '@/contracts';
 import { CharacterPower, CharacterTrait, GetTotalMultiplierForTrait, IWeapon, WeaponTrait } from '@/interfaces';
 import axios from 'axios';
 import Vue from 'vue';
-import { mapGetters } from 'vuex';
+import { mapActions, mapGetters } from 'vuex';
```
Comment (kcper) [inline frontend/src/components/smart/EarningsCalculator.vue:148]: Added mapGetters, but I don't see it being used anywhere.

**Comment by Chriserus** [comment]
@ed3ath Are you still working on it?

**Comment by kcper** [comment]
PR with additional improvements: https://github.com/CryptoBlades/cryptoblades/pull/850

**Pull request closed without merging**